app/views/rubric_criteria/_rubric_criterion_levels.html.erb

```diff
@@ -1,11 +1,25 @@
+<% content_for :head do %>
+  <%= render partial: 'boot',
+             formats: [:js],
+             handlers: [:erb] %>
+<% end %>
+
 <div id='rubric_levels_top'></div>
+<div id="add_level">
+  <%= link_to t('rubric_criteria.level.add'),
+              '#',
+              onclick: 'add_level("rubric_form");' +
+                'return false;' %>
```

Comment:
You can merge this into the previous line: `add_level("rubric_form"); return false;` in one string.